src/overlays/actors/ovl_Dm_Char08/z_dm_char08.c

```diff
@@ -229,13 +232,16 @@ void DmChar08_Init(Actor* thisx, PlayState* play2) {
         case SCENE_KONPEKI_ENT:
             this->unk_1FF = 2;
             this->animIndex = TURTLE_ANIM_FLOAT;
-            this->unk_203 = 0x63;
+            this->unk_203 = 99;
```

Comment:
99 is weird because normally -1 is used for `ANIM_NONE`, but 99 is commonly used for `CUEID_NONE`. But guess this particular actor decided to use 99 as NONE